wordpress-mobile/WordPress-iOS
# [TEST] Gutenberg - Disable featured image banner on iOS

Fixes #

To test:

1. Create a new page. 
2. Choose one of the blog templates. 
3. Notice that the featured image banner no longer exists. 

Before | After 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/117746768-89517e80-b1d2-11eb-896a-b04124f5df96.png" width="320"></kbd>  |       <kbd><img src="https://user-images.githubusercontent.com/1509205/117746773-8a82ab80-b1d2-11eb-81ff-0f88b895d738.png" width="320"></kbd>

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
